src/kili/services/export/format/kili/__init__.py

```diff
@@ -43,6 +45,15 @@ def _save_assets_export(self, assets: List[Asset], output_filename: Path):
                     with (labels_folder / f"{external_id}.json").open("wb") as output_file:
                         output_file.write(asset_json.encode("utf-8"))
             self.create_readme_kili_file(root_folder)
+
+            if self.download_media:
+                images_folder = base_folder / "images"
+                images_folder.mkdir(parents=True)
+                for asset in assets:
+                    img_path = images_folder / f'{asset["externalId"]}.jpg'
```

Comment:
extension should depend on the original content type. You can piggy back on the extension of the local file, for example with pattern matching `f"{external_id}.*"`